test/coverlet.core.tests/Reporters/CoberturaReporterTests.cs

```diff
@@ -120,5 +134,67 @@ public void TestReport()
             Assert.Equal(expectedMethodName, methodAttrs["name"]);
             Assert.Equal(expectedSignature, methodAttrs["signature"]);
         }
+
+        [Fact]
+        public void TestReportWithTwoDifferentDirectories()
+        {
+            CoverageResult result = new CoverageResult();
+            result.Identifier = Guid.NewGuid().ToString();
+
+            var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+
+            string absolutePath1;
+            string absolutePath2;
+
+            if (isWindows)
+            {
+                absolutePath1 = @"C:\projA\file.cs";
+                absolutePath2 = @"E:\projB\file.cs";
+            }
+            else
+            {
+                absolutePath1 = @"/projA/file.cs";
+                absolutePath2 = @"/projB/file.cs";
+            }
+
+            var classes = new Classes { { "Class", new Methods() } };
+            var documents = new Documents { { absolutePath1, classes }, { absolutePath2, classes } };
+
+            result.Modules = new Modules { { "Module", documents } };
+
+            CoberturaReporter reporter = new CoberturaReporter();
+            string report = reporter.Report(result);
+
+            var doc = XDocument.Load(new MemoryStream(Encoding.UTF8.GetBytes(report)));
```

Comment:
What do you think about 
```cs
List<string> rootPaths = doc.Element("coverage").Element("sources").Elements().Select(e => e.Value).ToList();
List<string> relativePaths = doc.Element("coverage").Element("packages").Element("package").Element("classes").Elements().Select(e => e.Attribute("filename").Value).ToList();
List<string> possiblePaths = new List<string>();
foreach (var root in rootPaths)
{
    foreach (string relativePath in relativePaths)
    {
        possiblePaths.Add(Path.Combine(root, relativePath));
    }
}
            
Assert.Contains(absolutePath1, possiblePaths);
Assert.Contains(absolutePath2, possiblePaths);
```
This should be the expected alg of every reporter that can read a cobertura format...we try every permutation root+path and check if exists

Reply:
Yes I changed it that way.